### PR TITLE
haste-task-rollup: add rollup task

### PR DESCRIPTION
**Pull request by bodia-uz**
Usage example:

https://github.com/bodia-uz/haste-preset-playable/blob/v1.0.2/src/commands/build.js#L96-L108
+
https://github.com/bodia-uz/haste-preset-playable/blob/v1.0.2/config/rollup.config.js

### CODE REVIEW

File: packages/haste-task-rollup/test/intex.spec.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
```
Comment (ronami) [inline packages/haste-task-rollup/test/intex.spec.js]: I think you accidentally named this file `intex.spec.js`

File: packages/haste-task-rollup/test/intex.spec.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    await test.run(async ({ [taskPath]: rollup }) => {
+      await rollup({ configPath: test.files['rollup.config.js'].path });
+    });
+
+    expect(test.files['bundle1.js'].exists).toBe(true);
+    expect(test.files['bundle2.js'].exists).toBe(true);
+
+    test.cleanup();
+  });
+});
```
Comment (ronami) [inline packages/haste-task-rollup/test/intex.spec.js]: I think we should add another test for when bundling fails.

File: packages/haste-task-rollup/test/intex.spec.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+      'rollup.config.js': fromFixture('./fixtures/rollup.config.js'),
+    });
+
+    await test.run(async ({ [taskPath]: rollup }) => {
+      await rollup({ configPath: test.files['rollup.config.js'].path });
+    });
+
+    expect(test.files['bundle.js'].exists).toBe(true);
+
+    test.cleanup();
```
Comment (ronami) [inline packages/haste-task-rollup/test/intex.spec.js]: let's move this to `afterEach`s

File: packages/haste-task-rollup/test/intex.spec.js (hunk 1 above)
Comment (bodia-uz) [reply packages/haste-task-rollup/test/intex.spec.js]: what name will be better?

Comment (ranyitz) [reply packages/haste-task-rollup/test/intex.spec.js]: `index.spec.js`

Comment (bodia-uz) [reply packages/haste-task-rollup/test/intex.spec.js]: Ah😧
Yes. You are right. 
Thanks. Will fix

**Comment by bodia-uz** [comment]
PR updated with fixes and additional tests.

**Comment by bodia-uz** [comment]
Cool! Thanks! 🎉

**Pull request merged**